images/src/generate.ts

```diff
@@ -0,0 +1,39 @@
+import OpenAI from 'openai';
+
+type ImageSize = '1024x1024' | '256x256' | '512x512' | '1792x1024' | '1024x1792';
```

Comment:
If these are the only size options, I think we should do more validation around this and also let the model know in the parameter description that it has to be one of these sizes.

Reply:
done